File: JungnangWomen/app/src/main/java/kr/susemi99/jungnangwomen/models/WomenResourcesClassItem.java
package kr.susemi99.jungnangwomen.models;

import com.google.gson.annotations.SerializedName;

public class WomenResourcesClassItem
{
  @SerializedName("list_total_count")
  public int listTotalCount;

  @SerializedName("RESULT")
  public ResultItem result;

  @SerializedName("row")
  public RowItem[] rows;

}

File: JungnangWomen/app/src/main/java/kr/susemi99/jungnangwomen/model/WomenResourcesClassItem.kt
package kr.susemi99.jungnangwomen.model

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class WomenResourcesClassItem(
  @SerialName("list_total_count") val listTotalCount: Int,
  @SerialName("RESULT") val result: ResultItem,
  @SerialName("row") val rows: List<RowItem>
)
